app/views/users/registrations/edit.html.erb

```diff
@@ -70,6 +75,8 @@
 </div>
 
 <%= render partial: '2fa_modal' %>
+<%= render partial: 'device_revocation_modal' %>
```

Comment:
I would move this partial inside the manage_devices partial, because it's required for it to work.